redactor/widgets.py

```diff
@@ -42,6 +42,8 @@ def render(self, name, value, attrs=None):
     def _media(self):
         _min = '' if settings.DEBUG else '.min'
         js = (
+            'admin/js/vendor/jquery/jquery{}.js'.format(_min),
+            'admin/js/jquery.init.js',
```

Comment:
1. That seems to be a weak assumption that these paths (provided by Django) would never change
2. Why exactly the issue has appeared? It was fine on 1.11. What has changed? Maybe there's a better way to fix this, without depending on the first point?

Reply:
Ok, answering to my own concerns:
1. Django itself does the same: all widgets that require jquery (like Date/Time ones) simply list these two files in their media, so I believe it's fine.
2. Looks like this issue is not related to 2.0. It's not always reproducible.